server.go

```diff
@@ -50,7 +50,7 @@ func main() {
 		http.Error(w, "", http.StatusNotFound)
 	})
 
-	log.Printf("endpoints initialized.")
+	log.Printf("endpoints initialized. %d")
```

Comment:
Just a notice: your test changes to server.go were pushed as well. Aside from this, the code looks good to merge now. Thanks Alyssa!